torch/_torch_docs.py

```diff
@@ -4108,6 +4108,44 @@
 
 """)
 
+add_docstr(torch.randint,
+           r"""
+randint(low=0, high, sizes, out=None) -> Tensor
```

Comment:
Can't we pass a dtype object to randint?

Reply:
Yeah, I just tried to mimic randn and rand in documentation which also expect type argument (in ATEN) but don't mention it in documentation. (Probably take default tensor set in environment)
Should I add it in doc?

Reply:
Add it please. The doc for `randn` and `rand` needs to be and will be updated.

Reply:
Alright, Default is torch.float32 right?